### PR TITLE
grid edits

### PR DESCRIPTION
**Pull request by FriGo74**

### CODE REVIEW

File: templates/gateways.html [hunk 1]
```diff
@@ -3,21 +3,95 @@
...
+    </div>
+   -->
+<!--    ----------------------------------------------------------------------------------------------------------------------------> 
+
+<br></br>
+
+
+
+
+<!--
```
Comment (BlackTurtle123) [inline templates/gateways.html]: Why the deuplicated code between comments?

Comment (FriGo74) [reply templates/gateways.html]: I'm still working on that. working of laptop switching between screens is not working nicely. this <i> <em> error is just bs ....Google says i sonarcloud says em I will do less PR so my workflow doesn’t interfere

Comment (BlackTurtle123) [reply templates/gateways.html]: trust in sonarcloud, it's a lot used in IT-land I would say

Comment (FriGo74) [reply templates/gateways.html]: well the exception is for icons sonar cloud just don't recognize  it ...but its ok i changed it

**Pull request merged**